arch/risc-v/src/common/riscv_cpuindex.c

```diff
@@ -48,10 +50,14 @@
  *
  ****************************************************************************/
 
-int up_cpu_index(void)
+uintptr_t riscv_cpuindex(void)
```

Comment:
Let's rename to `riscv_mhartid()`. What do you think?

Reply:
Sure